portfolio/src/main/webapp/index.html

```diff
@@ -36,74 +46,89 @@ <h1 class="headerFont">Matthew Wong</h1>
+            </div>
+            <div>
+                <a href="https://github.com/MattWong-ca/BodyMassIndexCalculator"><img src="/images/BMICalcSS.png" width="242" 
+                    height="451" class="cardImage"></a>
+            </div>
+        </div>
+
+        <div class="card">
+            <div class="cardTitle">
+                <p class="fonts">Crypto Converter 💸</p>
+            </div>
+            <div>
+                <a href="https://github.com/MattWong-ca/ByteCoin"><img src="/images/ByteCoinSS.png" width="242"
+                    height="451" class="image" style="width:100%"></a>
+            </div>
+        </div>
+
+        <div class="card">
+            <div class="cardTitle">
+                <p class="fonts">Live Weather App 🌤️</p>
+            </div>
+            <div>
+                <a href="https://github.com/MattWong-ca/LiveWeatherApp"><img src="/images/ClimaSS.png" href="https://github.com/MattWong-ca/LiveWeatherApp" width="242" height="451"
+                    class="image" style="width:100%"></a>
+            </div>
+        </div>
+
+        <div class="card">
+            <div class="cardTitle">
+                <p class="fonts">Quizzler 📚</p>
+            </div>
+            <div>
+                <a href="https://github.com/MattWong-ca/MultipleChoiceQuiz"><img src="/images/QuizzlerSS.png" href="https://github.com/MattWong-ca/MultipleChoiceQuiz" width="242" height="451" class="image" style="width:100%"></a>
+            </div>
+        </div>
+
+        </div>
+
     </div>
     </br>
 
-    <!--Just For Fun-->
-    <div class="test">
-    <p class="headerFont">Just for Fun 😄</p>
-        
-        <!--Fun Facts-->
-        <p class="mediumFont"">Fun Fact:</p>
-        <p class="fonts">Click <a href="/hello">here</a> to see content created by server code. Click here to get a random message:</p>
-        <button class="fonts" onclick="addRandomGreeting()">Hello</button>
-        <div class="fonts" id="greeting-container"></div>
-
-        <!--Server Date-->
-        <p class="mediumFont"">Today is...</p>
-        <p class="fonts">Click this button to see the server's current date:</p>
-        <button onclick="showServerTime();" class=" fonts">Today is...</button>
-        <p class="fonts" id="date-container"></p>
-        <p class="fonts">Click <a href="/date">here</a> to see the server response directly.</p>
-        
-        <!--JSON--><!--client-side JavaScript and server-side Java-->
-        <p class="mediumFont"">JSON</p>
-        <p class="fonts">Click <a href="/experience">here</a> to see JSON. Click the button to see the server-side JSON:</p>
-        <button class="fonts" onclick="test();">JSON</button>
-        <p class="fonts" id="exp"></p>
+    <!--Learnings-->
+    <div class="wideDivContent">
+        <p class="headerFont">Learnings 📖</p>
+        <p class="mediumFont">Check out what I've been up to:</p>
+        <p class="fonts">• <a href="learnings.html">Summer '22</a></p>
```

Comment:
How about:
 1. Make the learnings page more accessible 
 2. Move the "contact me" and "form responses" functions to the Learning page - where all SPS projects are consolidated.

Reply:
Added!